packages/sdk/src/client.ts

```diff
@@ -696,10 +708,7 @@ export class Client
             )
         }
 
-        const streamId =
-            !channelId && spaceId
-                ? makeMediaStreamIdFromSpaceId(spaceId)
-                : makeUniqueMediaStreamId()
```

Comment:
looks like you stopped using makeMediaStreamIdFromSpaceId, can we delete the function in id.ts?

Reply:
removed